UI/src/view/initialPopupView.js

```diff
@@ -159,11 +159,11 @@ window.InitialPopupView = function(backend, location, userRole, places, assetTyp
 
                 '<div class="contentNoBackColor">' +
                     '<div>' +
-                        '<p>' + "Tielolaji" + '<span class="assetTypeText">' + defaultAsset + '</span>' + '</p>'+
+                        '<p>' + "Tietolaji" + '<span class="assetTypeText">' + defaultAsset + '</span>' + '</p>'+
```

Comment:
Joo, nää pitäis mieluummin tulla jostakin lang conf tiedostosta kuin kovakoodata tänne mut minkäs voi. Yksi kieli vaatimuksena, joten ei tavallaan haittaa.  Toivotaan että ruotsinkieliset kunnat ei ala vaatia kääntämistä.